---
plugins:
  mesh:
    type: MeshPlugin
    coloring:
      type: SemanticMeshColoring
  places_basis_points:
    type: BasisPointPlugin
    show_voxblox_connections: false
    draw_basis_points: true
    basis_point_scale: 0.1
    places_node_scale: 0.1
  khronos_objects:
    type: KhronosObjectPlugin
    dynamic_color_mode: CONSTANT # ID, SEMANTIC, CONSTANT
  agent_poses:
    type: PosePlugin
    num_to_skip: 0
# dynamic reconfigure settings
places_basis_points:
  graph:
    visualize: true
